src/sage/schemes/elliptic_curves/ell_curve_isogeny.py

```diff
@@ -3326,6 +3329,80 @@ def _composition_impl(left, right):
         return NotImplemented
 
 
+def compute_isogeny_bmss(E1, E2, l):
+    r"""
+    Compute the kernel polynomial of the unique normalized isogeny
+    of degree ``l`` between ``E1`` and ``E2``.
+
+    Both curves must be given in short Weierstrass form, and the
+    characteristic must be either `0` or no smaller than ``4*l+4``.
+
+    ALGORITHM: [BMSS2006]_, algorithm *fastElkies'*.
+
```

Comment:
```
no smaller than `4l+4`
```

```
algorithm *fastElkies*.
```
Why `'`?

Reply:
The paper contains two algorithms, `fastElkies` and `fastElkies'`. They are similar but work with slightly different inputs.

Reply:
OK. Thanks.